lms/static/js/instructor_dashboard/instructor_dashboard.js

```diff
@@ -50,6 +50,12 @@ such that the value can be defined later than this assignment (file load order).
 
     $activeSection = null;
 
+    var usesProctoringLegacyView = function () {
```

Comment:
Why name this function this way? Why not name the function `usesProctoringMFEView` and use the logic in a reverse way?